### PR TITLE
Fixes #6440: We can't use é in the generic methods parameters

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6440

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated

File: tools/ncf.py [hunk 1]
```diff
@@ -147,7 +146,7 @@ def parse_bundlefile_metadata(content, bundle_type):
         else:
           res[tag] = match.group(1)
 
-    match = re.match("[^#]*bundle\s+agent\s+(\w+)(\(([^)]+)\))?.*$", line)
+    match = re.match("[^#]*bundle\s+agent\s+([a-zA-Z_]+)(\(([^)]+)\))?.*$", line, flags=re.UNICODE)
```
Comment (VinceMacBuche) [inline tools/ncf.py]: It misses '0-9' in the regexp

**Comment by VinceMacBuche** [comment]
@peckpeck Just this little change and you can self merge the PR

**Comment by peckpeck** [comment]
PR updated

File: tools/ncf.py [hunk 2]
```diff
@@ -147,7 +146,7 @@ def parse_bundlefile_metadata(content, bundle_type):
         else:
           res[tag] = match.group(1)
 
-    match = re.match("[^#]*bundle\s+agent\s+(\w+)(\(([^)]+)\))?.*$", line)
+    match = re.match("[^#]*bundle\s+agent\s+([0-9a-zA-Z_]+)(\(([^)]+)\))?.*$", line, flags=re.UNICODE)
```
Comment (ncharles) [inline tools/ncf.py]: why not \w anymore ? is it to prévent accents in bundle names ? (this is legal in cfengine)

**Comment by ncharles** [comment]
Oh, this change doesn't seem to work properly on Python 2.6 :(

**Pull request merged**